static/default/scripts/initSparnatural.js

```diff
@@ -61,4 +63,22 @@ sparnatural.addEventListener('componentLoaded',(e)=>{
   dropDown.addEventListener('loadQuery',(e)=>{
     sparnatural.loadQuery(e.detail.query)
   })
-})
\ No newline at end of file
+})
+
+document.getElementById('export').onclick = function() {
+  var jsonString = JSON.stringify(
+      JSON.parse(document.getElementById('query-json').value),
+      null,
+      2
+    );
+  $('#export-json').val(jsonString);
+  $('#exportModal').modal('show');       
+}
+
+
+document.getElementById('stored-queries').onchange = function() {
```

Comment:
What is this stored-quries? I don't see any element with this Id?

Reply:
This is a remainer of previous integration of sample queries dropdown. I have removed it.